bower.json

```diff
@@ -25,5 +25,8 @@
     "angular": "~1.2.18",
     "angular-mocks": "~1.2.18",
     "angular-ui-tinymce": "latest"
+  },
+  "resolutions": {
```

Comment:
I saw that the other PR's have their builds failing because there are multiple available angular versions. So I had to add this resolutions{} group to specify which one the project needs: https://travis-ci.org/jeff-collins/ment.io/builds/142782617

Probably the build tool did not detect that in the past but now it does...